src/net_processing.cpp

```diff
@@ -270,12 +270,6 @@ struct CNodeState {
     const CService address;
     //! Whether we have a fully established connection.
     bool fCurrentlyConnected;
-    //! Accumulated misbehaviour score for this peer.
-    int nMisbehavior;
-    //! Whether this peer should be disconnected and marked as discouraged (unless it has the noban permission).
-    bool m_should_discourage;
-    //! String name of this peer (debugging/logging purposes).
-    const std::string name;
```

Comment:
It absolutely should. When I wrote the original branch, I moved `name` to `Peer` as well, but since then, I removed all uses of `name` in #19472 and #19583 but failed to remove it as a member. I've now separated its removal into a separate commit.